tools/validators/instance_validator/validate/telemetry_warning.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Container for a telemetry validation error."""
+
+class TelemetryWarning(object):
```

Comment:
do we want to link into the findings model here?

Reply:
yes want to refactor and merge TelemetryWarning and TelemetryError into TelemetryReportElement? and we can have two or more levels or messages: warning, errors (for now)